CONTRIBUTING.md

```diff
@@ -119,4 +119,4 @@ Support this project with your organization. Your logo will show up here with a
 <a href="https://opencollective.com/osca/organization/6/website"><img src="https://opencollective.com/osca/organization/6/avatar.svg"></a>
 <a href="https://opencollective.com/osca/organization/7/website"><img src="https://opencollective.com/osca/organization/7/avatar.svg"></a>
 <a href="https://opencollective.com/osca/organization/8/website"><img src="https://opencollective.com/osca/organization/8/avatar.svg"></a>
-<a href="https://opencollective.com/osca/organization/9/website"><img src="https://opencollective.com/osca/organization/9/avatar.svg"></a>
\ No newline at end of file
+<a href="https://opencollective.com/osca/organization/9/website"><img src="https://opencollective.com/osca/organization/9/avatar.svg"></a>
```

Comment:
@BolajiAyodeji I didn't change anything I think during the time of creating this PR the main repo contributors got updated and this change reflected here.

Reply:
I don't think so, no updates have been made but it's all good